glean-core/rlb/src/glean_metrics.rs

```diff
@@ -0,0 +1,10 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// ** IMPORTANT **
+//
+// This file is *temporary*, it will be generated by 'glean-parser'
```

Comment:
Only sort of. We generate the `pings.rs`, we still need to include it somewhere.
So `glean_metrics.rs` won't go away fully.